README.md

```diff
@@ -10,11 +10,16 @@ can then perform whatever actions it wants.
 `snare` requires three command-line arguments to be specified:
 
 ```
-Usage: snare -p <port> -r <repo-programs-dir> -s <secrets-path>
+Usage: snare [-e email] [-j <maxjobs>] -p <port> -r <reposdir> -s <secretspath>
```

Comment:
`<max-jobs>` might be more aesthetically pleasing? Similarly with other double-barrelled argument names.

Reply:
Fixed in https://github.com/softdevteam/snare/pull/7/commits/d468f70cf288a13b8556b296a41c3845cbd195e5.